docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md

```diff
@@ -0,0 +1,146 @@
+---
+id: troubleshoot-zeebe-connection
+title: "Troubleshooting Zeebe Connection Issues"
+sidebar_label: "Zeebe Connection"
+---
+
+:::note
+Web Modeler Self-Managed is available to [enterprise customers](../../../../reference/licenses.md#web-modeler) only.
+:::
+
+You try to connect (i.e., to deploy) to a remote Zeebe cluster and Web Modeler tells you
+
+- the entered cluster endpoint URL should "point to a running Zeebe cluster."
+- an unknown error occurred and to "check Zeebe cluster status."
+
+To resolve this issue, check if you can connect to Zeebe through another client, i.e.,
+[`zbctl`](../../../../apis-tools/cli-client/index.md).
+If that doesn't work, resolve the general connection issue first (see for example [the platform deployment
+troubleshooting section](https://docs.camunda.io/docs/self-managed/platform-deployment/troubleshooting/)).
+If that works, further debug your Zeebe connection with the help of the information stated below.
```

Comment:
Which information stated below, to be clear?

Reply:
Basically everything below this sentence 🙃